internal/configure/topology/dc_get.go

```diff
@@ -236,6 +240,9 @@ type (
 		ChunkfilePoolRootDir  string // /curvebs/chunkserver/data
```

Comment:
missing `WalfilePoolRootDir` definition here, else the `layout.WalfilePoolRootDir` is an empty string.

Reply:
the same as to `WalfilePoolDir` and `WalfilePoolMetaPath`.

Reply:
fixed